scalameta/io/js/src/main/scala/scala/meta/io/PlatformIO.scala

```diff
@@ -0,0 +1,15 @@
+package scala.meta.io
+
+import java.io.InputStream
+import java.nio.charset.Charset
+
+object PlatformIO {
+  def workingDirectory: AbsolutePath = AbsolutePath(fileSeparator)
+  def read(stream: InputStream, charset: Charset): String =
+    throw new IllegalStateException("Reading from java.io.InputStream is not supported in JavaScript.")
```

Comment:
Thanks! `scala.io.Source.fromStream` failed to link so I simply assumed it was not possible. I've moved the method out of the `PlatformIO` object into an `InputStreamIO` object in the `shared/` folder.